pkgs/applications/audio/cantata/default.nix

```diff
@@ -54,12 +54,27 @@ let
   fstat = x: fn:
     "-DENABLE_${fn}=${if x then "ON" else "OFF"}";
 
-  fstats = x:
-    map (fstat x);
-
+  withHttpsSupport = true;
```

Comment:
Hm. It doesn't really make a difference, but I think I'd still prefer the old way of specifying `-DENABLE_HTTPS_SUPPORT` rather than having a "useless" boolean.

Everything else LGTM.

Reply:
How about dropping the variable and do this instead:

```
{ names = [ "HTTPS_SUPPORT" ]; enable = true; pkgs = [ ]; }
```

I'm a sucker for consistency.....

Reply:
Sure, that would work too.